api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -34,6 +34,50 @@ const (
 	PrivateDNSZoneModeNone string = "None"
 )
 
+// UpgradeChannel determines the type of upgrade channel for automatically upgrading the cluster.
+type UpgradeChannel string
+
+const (
+	// UpgradeChannelNodeImage automatically upgrades the node image to the latest version available.
+	// Consider using nodeOSUpgradeChannel instead as that allows you to configure node OS patching separate from Kubernetes version patching.
+	UpgradeChannelNodeImage UpgradeChannel = "node-image"
```

Comment:
```suggestion
	UpgradeChannelNodeImage UpgradeChannel = "node-image"
	
```
Please add a newline between variable declarations!

Reply:
ack